core/tests/snapshot-recovery-test/tests/snapshot-recovery.test.ts

```diff
@@ -370,7 +370,9 @@ async function getExternalNodeHealth() {
         if (e instanceof FetchError && e.code === 'ECONNREFUSED') {
             displayedError = '(connection refused)'; // Don't spam logs with "connection refused" messages
         }
-        console.log('Request to EN health check server failed', displayedError);
+        console.log(
+            `Request to EN health check server failed ${displayedError}, in CI you can see more details in "Show snapshot-creator.log logs" and "Show contract_verifier.log logs" steps`
```

Comment:
Nit: This line is overly long, normally we try respect 120 characters line limit.

Reply:
corrected